release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3619,3 +3619,28 @@ $ oc adm release info 4.11.43 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-11-44"]
+=== RHSA-2023:3915 - {product-title} 4.11.44 bug fixes and security update
+
+Issued: 2023-07-05
+
+{product-title} release 4.11.44, which includes security updates, is now available. This update includes a Red Hat security bulletin for customers who run {product-title} in FIPS mode. For more information, see link:https://access.redhat.com/security/vulnerabilities/RHSB-2023-001[RHSB-2023:001]. Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:3915[RHSA-2023:3915] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3914[RHSA-2023:3914] advisory.
```

Comment:
```suggestion
{product-title} release 4.11.44, which includes security updates, is now available. This update includes a Red Hat security bulletin for customers who run {product-title} in FIPS mode. For more information, see link:https://access.redhat.com/security/vulnerabilities/RHSB-2023-001[RHSB-2023:001]. 

Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:3915[RHSA-2023:3915] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3914[RHSA-2023:3914] advisory.
```

Reply:
Just to add consistency across versions.

Reply:
Done.